Dockerfile

```diff
@@ -1,15 +1,40 @@
-FROM rust:slim AS builder
-RUN apt-get update && apt-get -y upgrade && apt-get install -y apt-utils
-RUN apt-get install -y libssl-dev pkg-config
-COPY . iroha/
-WORKDIR iroha
+ARG BASE_IMAGE=ubuntu:20.04
+FROM $BASE_IMAGE AS builder
```

Comment:
Why can't we use Rust slim as builder?

Reply:
It is updated slowly and there is only debian. I personally consider ubuntu more stable as it has some libraries which debian doesn't have.

But anyway it can changed later, but for rust-slim you won't be able to change base image

Reply:
Rust also has an option to choose Alpine Linux as a base image

Reply:
Ok, if the update speed leads to build inconsistency let's switch to ubuntu